apps/hash-api/src/graphql/create-apollo-server.ts

```diff
@@ -79,55 +80,55 @@ export const createApolloServer = ({
     // @todo: we may want to disable introspection at some point for production
     introspection: true,
     debug: true, // required for stack traces to be captured
+    // See https://www.apollographql.com/docs/apollo-server/integrations/plugins/#request-lifecycle-event-flow
     plugins: [
       {
         requestDidStart: async (ctx) => {
           ctx.logger = ctx.context.logger as Logger;
-          const startedAt = performance.now();
+
+          const startTimestamp = performance.now();
 
           return {
-            async didEncounterErrors(errorContext) {
-              const user: Express.Request["user"] = errorContext.context.user;
+            didResolveOperation: async (didResolveOperationCtx) => {
+              const operationName = didResolveOperationCtx.operationName;
+              if (operationName) {
+                // transaction.setName(operationName, "route");
```

Comment:
Is this mean to be here?

Reply:
Thanks, that was an old fragment which is not required anymore: bae93498d9c2e458cee6acfe80d71ad0e7415a7b